### DISCUSSION TITLE
argoCD  read: connection reset by peer

### DISCUSSION THREAD

**Discussion by myu2018** [argoproj/argo-cd#13555] [Q&A]
Hi! Folk,

I upgraded my self-hosted argoCD to v2.6.7+5bcd846 w/o issue. 

But I notice that whenever I was trying to view the detail of a deployed object in argoCD application, I always got the error 

"Unable to load data: error getting cached app managed resources: read tcp 10.40.4.121:52714->10.0.134.84:6379: read: connection reset by peer".

I did further research and found following logs in argocd-server's pod:
...
redis: 2023/05/11 20:30:58 pubsub.go:159: redis: discarding bad PubSub connection: read tcp 10.40.4.121:52528->10.0.134.84:6379: read: connection reset by peer
...

It appears that the argoCD server for some reason failed to talk to argocd-redis service?

It seems that, during the upgrade, somehow the redis setting was messed up?

Appreciate help from experts. 

Thanks.

**Comment by Jose-Matsuda** [comment]
Have you been able to find anything out regarding this?
